rasa/api.py

```diff
@@ -121,7 +120,9 @@ def train(
             model_to_finetune=model_to_finetune,
             finetuning_epoch_fraction=finetuning_epoch_fraction,
         ),
-        loop,
+        # TODO: asyncio.run doesn't let you set a custom loop,
```

Comment:
This is the only call I could find to the old `run_in_loop` function that actually let you specify a specific `loop` - and none of the invocations of this use it. However, I'm pretty sure this function (`train`) is a public API so removing a parameter might require some delicate handling. What do you think

Reply:
Good point - I think dropping is fine as long as point that out in the changelog 👍🏻